Eagle.gitignore

```diff
@@ -31,14 +34,19 @@ eagle.epf
 *.drl
 *.gpi
 *.pls
+*.ger
+*.gpi
```

Comment:
Note this was a duplicate entry. c.f https://github.com/github/gitignore/pull/2741